index.js

```diff
@@ -1 +1,13 @@
 // code your solution here
+
+const superbowlWin = record => {
+  const yearWon = record.find(findWin)
+  if (typeof yearWon === 'undefined'){
+    return yearWon
+  } else {
+    return yearWon.year
+  }
+}
+
+
+const findWin = element => element.result === "W"
```

Comment:
What's a better name for `element`? What IS the element??

Reply:
I think result would have been better here. I must have forgotten to scrub back through my code and name all my variables.

Reply:
Hm, I don't think `result` would be better. You'd then be calling `result.result` which is a little... ambiguous. Any other ideas? I mean really... what IS the element? What would that argument contain??